UPGRADE.md

```diff
@@ -0,0 +1,27 @@
+# UPGRADE FROM `v1.2.0` TO `v1.2.1`
```

Comment:
im not sure if we shouldn't release it as 2.0 due to lack of backward compatibility